modules/network-flow-matrix.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/configuring-firewall.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-flow-matrix_{context}"]
+= {product-title} network flow matrix
+
+The network flow matrix describes the ingress flows to {product-title} services.
+The network information in the matrix is accurate for both bare-metal and cloud environments.
+Use the information in the network flow matrix to help you manage ingress traffic.
+You can restrict ingress traffic to essential flows only to improve network security.
```

Comment:
```suggestion
You can restrict ingress traffic to only essential flows to improve network security.
```
just a suggestion as the way it reads is somehow awkward

Reply:
Thanks - actually I just removed "only" and I think the meaning remains the same